src/NUnitFramework/framework/Constraints/EqualConstraint.cs

```diff
@@ -323,6 +323,17 @@ public EqualConstraint Using<T>(IComparer<T> comparer)
         }
 
         /// <summary>
+        /// Flag the constraint to use the supplied (T, T) => bool object.
+        /// </summary>
+        /// <param name="comparer">The (T, T) => bool object to use.</param>
```

Comment:
`(T, T) => bool` was my pseudo-C# shorthand, perhaps influenced by [Tyler's proposal](https://github.com/dotnet/csharplang/issues/269), but I wonder if it wouldn't be more in line with the other overloads' XML documentation to be more language-agnostic and say `boolean-returning delegate` instead of `(T, T) => bool object`?

(2 places)